src/wallet/wallet2.h

```diff
@@ -681,6 +681,7 @@ namespace tools
 
     void commit_tx(pending_tx& ptx_vector);
     void commit_tx(std::vector<pending_tx>& ptx_vector);
+    std::string save_tx(const std::vector<pending_tx>& ptx_vector) const;
```

Comment:
I'm inclined to even rename it to `save_tx_to_str` and `save_tx_to_file` for clarity, but admittedly it's subjective.